superbench/analyzer/file_handler.py

```diff
@@ -221,3 +222,29 @@ def output_lines_in_html(lines, output_path):
             f.writelines(html_str)
     except Exception as e:
         logger.error('FileHandler: html_data_output - {}'.format(str(e)))
+
+
+def merge_column_in_excel(ws, row, column):
```

Comment:
So the input data should make sure that the same contents should be continuous? Should we add some check?

Reply:
I think this function is designed to merge the cells for continuous same contents rather than merge cells for same contents. if the contents are not continuous but the same, it means they don't want to be merged, that depends on how the input data want to display.